packages/packages/theme-builder/src/init.tsx

```diff
@@ -0,0 +1,13 @@
+import { registerAction } from '@elementor/top-bar';
+import useActionProps from './hooks/use-action-props';
+
+export default function init() {
+	registerTopBarMenuItems();
+}
+
+function registerTopBarMenuItems() {
+	registerAction( 'main', {
+		name: 'open-theme-builder',
+		useProps: useActionProps,
+	} );
```

Comment:
```suggestion
	registerAction( 'main', {
		name: 'open-theme-builder',
		props: {
			icon: ThemeBuilderIcon,
			title: __( 'Theme Builder', 'elementor' ),
			onClick: () => runCommand( 'app/open' ),
		}
	} );
```

Reply:
@Nevoss - For now I'll keep it as it is for the unit testing until we'll decide on a way for testing it.